js/views/settingsVw.js

```diff
@@ -560,8 +559,11 @@ module.exports = Backbone.View.extend({
     localStorage.setItem('NSFWFilter',  this.$("#generalForm input[name=nsfw]:checked").val());
 
     saveToAPI(form, this.userModel.toJSON(), self.serverUrl + "settings", function(){
-      "use strict";
-      messageModal.show(window.polyglot.t('saveMessages.Saved'), "<i>" + window.polyglot.t('saveMessages.SaveSuccess') + "</i>");
+      app.statusBar.pushMessage({
+        type: 'confirmed',
+        msg: '<i>' + window.polyglot.t('saveMessages.SaveSuccess') + '</i>'
```

Comment:
You can include html in the message. We probably shouldn't be too crazy with it, but simple inline stuff shouldn't be a problem at all.